### ISSUE TITLE
Enable apps panel in Nextcloud

### ISSUE THREAD

**Issue by lifeboy** [ddavness/power-mailinabox#111]
I've been searching high and low to find how one can get to the apps panel in nextcloud, but so far it has eluded me.  The only thing I can find is a reference to having to be an administrator of nextcloud to see the apps panel.  

Question: Is a pmiab admin also a nexcloud admin?  If so, how does one enable to apps panel please?

If nextcloud needs a special admin to see the apps panel, where is that user?

Some insights and advice would be much appreciated.

**Comment by nanotronic** [comment]
Add your user to the admin group with
https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/occ_command.html#group-commands

**Comment by lifeboy** [comment]
On Mon, 6 Feb 2023 at 09:33, nanotronic ***@***.***> wrote:

> Add your user to the admin group with
>
> https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/occ_command.html#group-commands
>
Thanks for the link.  However, what is the relationship between a P-MIAB
user (ie. ***@***.***) and the a PAM Linux user?  My admin account
in P-MIAB does not have a PAM user.

**Comment by nanotronic** [comment]
Yes, the admin user seems to be different ones. For nextcloud there will be a root user created: 
https://github.com/ddavness/power-mailinabox/blob/main/setup/nextcloud.sh#L341
Changing the password for root with `occ user:resetpassword root` worked, but logging in was still not possible. Therefore, I added another user to the admin group. The App Panel is then accessible with this user.

**Comment by lifeboy** [comment]
Ok, I was too quick to post above:

```
# sudo -u www-data php occ group:adduser admin roland
user not found

```

So, if I list the users...

# sudo -u www-data php occ user:list
  - mariesa@mailbox.com: mariesa@mailbox.com
  - pablice@mailbox.com: pablice@mailbox.com
  - roland@mailbox.com: roland@mailbox.com
  - root: root
# sudo -u www-data php occ group:adduser admin roland@mailbox.com
An unhandled exception has been thrown:
TypeError: OCA\Circles\Tools\Model\Request::setUsedProtocol(): Argument #1 ($protocol) must be of type string, bool given, called in /usr/local/lib/owncloud/apps/circles/lib/Tools/Model/Request.php on line 295 and defined in /usr/local/lib/owncloud/apps/circles/lib/Tools/Model/Request.php:167

So maybe it's the "@" that's throwing this off course...

# sudo -u www-data php occ group:adduser admin "roland@mailbox.com"

Yes, indeed, now the user is added to admins and I can access the apps panel and other admin features.

**Comment by ddavness** [comment]
**For future reference:** MIAB shipped with a script that can be used for this: `tools/owncloud-unlockadmin.sh`.

https://github.com/ddavness/power-mailinabox/blob/main/tools/owncloud-unlockadmin.sh
